oneview_redfish_toolkit/multiple_oneview.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Python libs
+import logging
+
+# 3rd party libs
+from hpOneView.exceptions import HPOneViewException
+
+# Modules own libs
+from oneview_redfish_toolkit import authentication
+from oneview_redfish_toolkit import connection
+
+# Globals vars:
+#   globals()['map_resources_ov']
+
+
+NOT_FOUND_ERROR = ['RESOURCE_NOT_FOUND', 'ProfileNotFoundException',
```

Comment:
We should map this errors at ResponseBuilder class to keep the consistency with the handler of HPOneViewException.

Reply:
How about we place it in **oneview_redfish_toolkit/api/errors.py** ?

Reply:
Looks good, but my point is to put this errors in [ResponseBuilder](https://github.com/HewlettPackard/oneview-redfish-toolkit/blob/master/oneview_redfish_toolkit/blueprints/util/response_builder.py#L24), too.
To avoid that some exceptions do not be handled.
Because, for now, we are handling just error_code `RESOURCE_NOT_FOUND` from Oneview, right?! But we can have `['RESOURCE_NOT_FOUND', 'ProfileNotFoundException', DFRM_SAS_LOGICAL_JBOD_NOT_FOUND]` too.

Reply:
Yes you right, I was just thinking if this array should be on ResponseBuild or not. But I agree with you, wherever it will be, we must change ResponseBuilder to use it as well.